manifests/qpid.pp

```diff
@@ -1,61 +1,11 @@
 # @summary Katello qpid Config
 #
-# @param interface
-#   The interface to listen on
-#
-# @param wcache_page_size
-#   The size (in KB) of the pages in the write page cache
-#
-class katello::qpid (
-  String $interface = 'lo',
-  Integer[0, 5000] $wcache_page_size = 4,
-) {
-  include katello::params
-
-  $agent_event_queue_name = $katello::params::agent_event_queue_name
-
-  if $katello::params::enable_katello_agent {
-    include certs
-    include certs::qpid
-
-    $_qpid_ensure = 'present'
-    $ssl = true
-    $ssl_cert_db = $certs::qpid::nss_db_dir
-    $ssl_cert_password_file = $certs::qpid::nss_db_password_file
-    $ssl_cert_name = $certs::qpid::nss_cert_name
-
-    Class['certs', 'certs::qpid'] ~> Class['qpid']
-  } else {
-    $_qpid_ensure = 'absent'
-    $ssl = false
-    $ssl_cert_db = undef
-    $ssl_cert_password_file = undef
-    $ssl_cert_name = undef
```

Comment:
Did your testing find that we do not need these three `undef` statements in order to fully remove qpid?

Reply:
No, I just checked that the parameters had the Optional type -- can the `ensure => 'absent'` fail if these are unspecified?

Reply:
I should be able to test this scenario using the module PRs feature from forklift, actually